src/main/java/com/cinema/controllers/AuthenticationController.java

```diff
@@ -0,0 +1,24 @@
+package com.cinema.controllers;
+
+import com.cinema.model.dto.UserRequestDto;
+import com.cinema.security.AuthenticationService;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/cinema")
```

Comment:
It might be a good idea not to add extra words to our URL. You could just have `http://localhost:8080/registration`. Then you can simply remove the @RequestMapping annotation from here

Reply:
I just thought - I might have on my local host another project (like internet shop) and it might have registration page as well ...